### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by yavora** [ggiraldez/crystal-gl#1]
Hi, are you still working on this shard? :c

**Comment by asterite** [comment]
Not really. Someone should make some nice, complete, nicely wrapped SDL + OpenGL bindings some day...

**Comment by ggiraldez** [comment]
Yep, this is sort of abandoned, at least for the moment. Feel free to fork and extend if you find something useful.

**Comment by girng** [comment]
@asterite don't want to let you down asterite.. please check gitter we are trying something.

also https://github.com/calebuharrison/LibGL looks really promising. i think we should use that a base and start working on primitives (draw_char, draw_texture), etc. godot's repo has all these done in opengl already, so we can get a good head start
